File: ejercicios-py/ejercicio8.py
import random

# Generar un array de números aleatorios entre 1 y 100
array = [random.randint(1, 100) for _ in range(10)]

# Calcular la sumatoria de los elementos del array
sumatoria = sum(array)

# Imprimir los resultados en GitHub Actions
print(f"La sumatoria del array es: {sumatoria}")

File: ejercicios-js/ejercicio8.js


public= class ArraySum {
    public= static void main(String[] args) {
        int[] numbers = {1, 2, 3, 4, 5};
        int sum = calculateSum(numbers);
        System.out.println("La suma de los elementos del array es: " + sum);
    }

    public= static int calculateSum(int[] array) {
        int sum = 0;
        for (int number : array) {
            sum += number;
        }
        return sum;
    }
}